build/get_true_labels.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import numpy as np
+import warnings
+warnings.simplefilter('ignore')
+
+def load_df():
+    """Read in processed data."""
+    print("Reading in processed data.")
+    df = pd.read_hdf('./data/processed/bexar_processed.h5')
+    return df
+
+def get_true_labels():
+    """Return dataframe with only properties with true labels."""
+    df = load_df()
+
+    # Fill na values for price_psf with 0 (means it's an empty lot)
+    df['price_psf'] = df['price_psf'].fillna(0)
+
+    # Filter out wonky situations (erroneous property encoding, sq footage of 1, etc.)
+    trim_prop_df = df[(df.price_psf<df.price_psf.quantile(.999))]
```

Comment:
best practice is to avoid hard-coding any variable values in code. an improvement over your current set-up is to have variables for quantile .999, top 30 in the below line, etc. defined in this file. but the best practice would be to have them defined in a yaml file. Therefore, if anyone wanted to re-purpose your code, apply it to a different data-set, or change any of these variables, there would be an easy way to do it that didn't require knowing python.